phpBB/phpbb/extension/metadata_manager.php

```diff
@@ -47,6 +53,25 @@ public function __construct($ext_name, $ext_path)
 		$this->ext_name = $ext_name;
 		$this->metadata = array();
 		$this->metadata_file = $ext_path . 'composer.json';
+
+		// Initialize validations
+		$this->validations = array(
```

Comment:
Agreed.  Didn't do it as I was not sure it was ok for the minimum PHP version.
Will do it tomorrow.